test/e2e/item_data_source_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"regexp"
 	"testing"
 
+	op "github.com/1Password/connect-sdk-go/onepassword"
```

Comment:
`[nit]`
As convention we usually sort imports in 3 groups in the following order:
1. Standard library
2. 3d party dependencies
3. Packages from current project